src/SearchResults.svelte

```diff
@@ -63,21 +64,24 @@
      * Use the specified format and ATT&CK object to place a snippet on the
      * clipboard.
      */
-    async function copyFormat(format, object, resultIdx, formatIdx) {
+     async function copyFormat(format, object, resultIdx, formatIdx) {
```

Comment:
Looks like one extra space added in front of `async` -- should be indented 4 spaces. Please double check.

Reply:
Also if you use Visual Studio Code, install the Svelte for VS Code plugin. It gives better intellisense and can autoformat code for you to avoid minor formatting issues.

![Screen Shot 2023-03-24 at 1 21 39 PM](https://user-images.githubusercontent.com/320904/227596672-6a778c40-64c6-4d93-aaf2-b52bc507ada0.png)